k8s/upgrades/README.md

```diff
@@ -22,7 +22,7 @@ If already using CSPC pools and cStor CSI volumes they can be upgraded from `1.1
 
 This document describes the steps for the following OpenEBS Upgrade paths:
 
-- Upgrade from 1.6.0 or later to a newer release up to 2.11.0
+- Upgrade from 1.6.0 or later to a newer release up to 2.12.0
```

Comment:
@shubham14bajpai - i forgot this. why are we saying 1.6.0 here. Is it become prior releases are deprecated. In that case can we update this section to say 1.8.0 to match with above changes of community pipelines.

Reply:
Yes this is oldest version we test in community pipelines.